README.md

```diff
@@ -86,16 +94,13 @@ If so then:
 
 ```yaml
 gitlab:
-  # You can also set in your environment GITLAB_URL
   url: https://gitlab.yourcompany.com
-  # You can also set in your environment GITLAB_TOKEN
   token: "<private token of an admin user>"
-  api_version: 4
 
-group_settings:
-  my-group:
+projects_and_groups:
+  my-group":
     deploy_keys:
-      a_friendly_deploy_key_name:  # this name is only used in GitLabForm config
+      a_friendly_deploy_key_name: # this name is only used in the config
```

Comment:
I prefer never to have code and comment on the same line so that if you modify comments, you don't alter lines with code. you will feel the difference when you want to `git blame` to find the commit that introduced the change for code.

Reply:
Thanks, your advice definitely makes sense, @glensc ! I do care about `git blame` (even so much that I discouraged people from making formatting-only changes because I was not aware of the fact that [it can ignore whitespace changes and moving things around](https://coderwall.com/p/x8xbnq/git-don-t-blame-people-for-changing-whitespaces-or-moving-code)...) so I will try to fix this config and apply this in other places from now on. :)

Reply:
also, recommended reading:
- https://www.freshconsulting.com/atomic-commits/
- https://chris.beams.io/posts/git-commit/